content/cloud/noops-sql-db.md

```diff
@@ -38,15 +42,13 @@ Once you have access to the private beta, please ensure you have [Spin CLI](./cl
 *Service Limitations*
 * To run a SQL statement (for migration and schema creation), use the `spin cloud sqlite list` and `spin cloud sqlite execute` commands. The equivalent of `spin up --sqlite` for `spin deploy` does currently not exist in Cloud. [See this section for more info](#tables-and-data-in-fermyon-cloud).
 
-* Fermyon **Cloud** only supports databases named "default" at this time.
-
 ## Creating a New Spin Application
 
 If you already have a Spin application, you can skip this step. If you do not have a Spin application locally you will need to [create one](/spin/quickstart#create-your-first-application).
 
 ## Grant SQLite Permission
 
-To tell Spin that we want to use SQLite storage, we only need to [grant SQLite permission to a component in the application’s manifest](/spin/sqlite-api-guide#granting-sqlite-database-permissions-to-components) (the `spin.toml` file). For example:
+To tell Spin that we want to use SQLite storage, we only need to [grant SQLite permission to a component in the application’s manifest](/spin/sqlite-api-guide#granting-sqlite-database-permissions-to-components) (the `spin.toml` file) by supplying a [label](labels.md). For example:
```

Comment:
Looks like there is no page called `/cloud/label` - that is the broken link we should switch to absolute paths instead of relying on relative paths?

Reply:
Thanks @karthik2804 
Good spotting.

Reply:
Update coming in next commit/push.